sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/VectorizedHashMapGenerator.scala

```diff
@@ -75,9 +77,14 @@ class VectorizedHashMapGenerator(
           }
         }.mkString("\n").concat(";")
 
+    val columnVector = if (!conf.offHeapColumnVectorEnabled) {
+      "OnHeapColumnVector"
+    } else {
+      "OffHeapColumnVector"
+    }
     s"""
-       |  private org.apache.spark.sql.execution.vectorized.OnHeapColumnVector[] batchVectors;
-       |  private org.apache.spark.sql.execution.vectorized.OnHeapColumnVector[] bufferVectors;
```

Comment:
Let's not bother about here too, this class is only used in benchmark AFAIK.

Reply:
I see. `"spark.sql.codegen.aggregate.map.vectorized.enable" = true` only in benchmark or test cases.